src/components/Breadcrumb/index.tsx

```diff
@@ -1,43 +1,49 @@
 import React from "react";
 import { Breadcrumb } from "antd";
-import { routes, paths } from "constants/routes";
+import { paths } from "constants/routes";
 import styled from "@emotion/styled/macro";
 import { uiColors } from "@leafygreen-ui/palette";
 import { H3, P1 } from "components/Typography";
 import { StyledRouterLink } from "components/styles/StyledLink";
 import { useBreadcrumbAnalytics } from "analytics";
+import { useGetUserPatchesPageTitleAndLink } from "hooks";
 
 const { blue } = uiColors;
 
 interface Props {
   versionId?: string; // only required when rendered on task page
   taskName?: string;
   patchNumber: number;
+  patchAuthor: string;
 }
 
 export const BreadCrumb: React.FC<Props> = ({
   versionId,
   taskName,
   patchNumber,
+  patchAuthor,
 }) => {
   const breadcrumbAnalytics = useBreadcrumbAnalytics();
   const patch = `Patch ${patchNumber}`;
-
+  const {
+    title: userPatchesPageTitle,
+    link: userPatchesPageLink,
+  } = useGetUserPatchesPageTitleAndLink(patchAuthor);
   return (
     <StyledBreadcrumb>
       <Breadcrumb.Item>
         <StyledP1>
           <StyledBreadcrumbLink
             id="bc-my-patches"
-            to={routes.myPatches}
+            to={userPatchesPageLink}
             onClick={() =>
               breadcrumbAnalytics.sendEvent({
                 name: "Click Link",
                 link: "myPatches",
```

Comment:
Do ya'll think we should adjust the analytics for this page to include the author userId?

Reply:
I don't think so, unless we care about seeing when people click on other people's patches vs their own